src/APIAuthor.ts

```diff
@@ -547,4 +547,26 @@ export class APIAuthor {
         onCompletion?: (err: Error)=> void): Promise<void>{
         return PromiseWrapper.voidPromise(new ThingOps(this, thingID).updateThingType(thingType), onCompletion);
     }
+
+    /** Get firmware version of specified target thing. If firmware versoin is not set, then null is returned.
```

Comment:
This might be trivial but someone might think that `then null is returned.` is `returned promise is null`.

What do you think about `null is passed as an argument of the returned promise`?

The statement which I suggest may be redundant. You can ignore this comment:)